tox.ini

```diff
@@ -1,17 +1,19 @@
 [tox]
 envlist =
-    py{36,37,38,39,310}-django-22
-    py{36,37,38,39,310}-django-32
-    py{38,39,310}-django-40
-    py{38,39,310}-django-main
+    py{36,37,38,39,310}-django22
+    py{36,37,38,39,310}-django32
+    py{38,39,310}-django40
+    py{38,39,310}-django41
+    py{38,39,310}-djangomain
 
 [testenv]
 passenv = *
 commands =
     coverage run manage.py test
 deps =
-    py{36,37,38,39,310}-django-22: Django>=2.2,<2.3
-    py{36,37,38,39,310}-django-32: Django>=3.2,<3.3
-    py{38,39,310}-django-40: Django>=4.0,<4.1
-    py{38,39,310}-django-main: https://github.com/django/django/archive/main.tar.gz
+    django22: Django>=2.2,<2.3
+    django32: Django>=3.2,<3.3
+    django40: Django>=4.0,<4.1
+    django41: Django>=4.1,<4.2
```

Comment:
@adamchainz After upgrading to python 3.10 and Django 4.1, it is creating a new migration file. Did you observe this?
Attaching a sample SS:
![Screenshot 2022-11-01 at 2 11 43 AM](https://user-images.githubusercontent.com/87464570/199106395-9a23f239-43d5-403e-bfb7-4ab5a676340f.png)

Reply:
Thank you @adamchainz!